global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Each of our projects should opt into which dotnet version they need
(it failed to load on my machine, as I've got a root global.json that defaults it to 3.1, like the build agents)